site/docs/connectors/integration/event-schema-integration-connector.md

```diff
@@ -87,7 +87,7 @@ Set the following variables with valid value:
 
 ## Running locally for testing
 
-You will need to have an OMAG platform with a configured metadata server and activated DataManager OMAS. You will also have a Docker client installed locally and a running Docker daemon.
+You will need to have an OMAG platform with a configured [Metadata Access Point](https://egeria-project.org/concepts/metadata-access-point/) and activated DataManager OMAS. You will also have a Docker client installed locally and a running Docker daemon.
```

Comment:
If you use the metadta access point then the server needs to be connected to a cohort containing at least on metadata access store.  If it can be either then use the term [metadata access server](https://egeria-project.org/concepts/metadata-access-server/)

Reply:
I changed that also in the concept of the Metadata Access Server as the description there lead to the assumption that a Metadata Access Store is a specialized Metadata Access Point instead of a specialized Metadata Access Server.

Reply:
Both are true ... when it comes to the java clients, metadata access store inherits from metadata access point - because it adds the ability to have a repository.   This is fine for technical people.  We use the metadata access server term to mean either for less technical audiences whether the concept of inheritance is not natural.